### PR TITLE
[ru] Add "Tightly Coupled Architecture"

### PR DESCRIPTION
**Pull request by hcz**
### Describe your changes
[ru] Add "Tightly Coupled Architecture" #3203

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/tightly-coupled-architecture.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Внедрять такую архитектуру обычно легче, чем более слабо связанные стили архитектуры,
+но это может сделать систему более уязвимой к каскадным сбоям.
+Также они, как правило, требуют координированных развертываний компонентов,
+что может снизить производительность разработчиков.
+
+Тесно связанную архитектуру можно назвать традиционным способом создания приложений.
+Хотя этот стиль и не соответствует практикам разработки [микросервисов](/microservices-architecture/),
+такая архитектура может быть полезна в определенных обстоятельствах.
+Её внедрение обычно быстрее и проще, и,
+как и [монолитные приложения](/monolithic-apps/), может ускорить начальный цикл разработки.
```
Comment (kirkonru) [inline content/ru/tightly-coupled-architecture.md:20]:
```suggestion
Тесно связанные архитектуры — традиционный способ создания приложений.
Хотя они не всегда соответствует лучшим практикам разработки [микросервисов](/microservices-architecture/),
такие архитектуры могут быть полезны в определенных обстоятельствах.
Как правило, они быстрее и проще в реализации и, 
подобно [монолитным приложениям](/monolithic-apps/), могут ускорить начальный цикл разработки.
```

**Review by kirkonru — COMMENTED** [review]
Thanks for the translation! Did some minor rephrasing...

**Review by shurup — APPROVED** [review]
LGTM! Thank you for your work!

**Review by kirkonru — APPROVED** [review]

**Comment by kirkonru** [comment]
@hcz Thanks for your contribution! Please note that you have to use **dev-ru** both as a source to create your own branch when editing/adding files and as a target one to merge your PR into.

**Pull request merged**